.github/workflows/on-remove.yml

```diff
@@ -125,6 +125,41 @@ jobs:
           SQUIDEX_CLIENT_ID: ${{ steps.setup.outputs.gp2-squidex-ci-client-id }}
           SQUIDEX_CLIENT_SECRET: ${{ secrets.GP2_SQUIDEX_CI_CLIENT_SECRET }}
 
+  crn-contentful-remove:
+    if: ${{ github.event.inputs.pr-number || github.event.pull_request.head.repo.full_name == 'yldio/asap-hub' }}
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/yldio/asap-hub/node-python-sq:be0ebbcbd0ff209d56070f09590c5d9622e8a6dc
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    environment: Branch
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup
+        id: setup
+        uses: ./.github/actions/setup-environment
+        with:
+          environment-name: Branch
+      - name: Contentful checks
```

Comment:
I think I would rename this to something like 'Get Contentful branch environment name' or something, that way I would know immediately what we are calling the contentful-checks action in this scenario

Reply:
We're getting the branch environment as well as whether we're on a branch environment. Will rename.

Reply:
Done.